app/db/base.py

```diff
@@ -3,21 +3,21 @@
 from sqlalchemy.orm import declarative_base
 from sqlalchemy.orm import sessionmaker
 
-IS_SQLITE = SQLALCHEMY_DATABASE_URL.startswith('sqlite')
+IS_SQLITE = SQLALCHEMY_DATABASE_URL.startswith("sqlite")
 
 if IS_SQLITE:
     engine = create_engine(
         SQLALCHEMY_DATABASE_URL,
         connect_args={"check_same_thread": False},
-        #echo=True
+        # echo=True
```

Comment:
Avoid technical comments that disable and toggle properties of code execution procecss.